deisctl/cmd/platform.go

```diff
@@ -11,12 +11,18 @@ import (
 
 // InstallPlatform loads all components' definitions from local unit files.
 // After InstallPlatform, all components will be available for StartPlatform.
-func InstallPlatform(b backend.Backend, checkKeys func() error) error {
+func InstallPlatform(b backend.Backend, checkKeys func() error, stateless bool) error {
 
 	if err := checkKeys(); err != nil {
 		return err
 	}
 
+	if stateless {
+		fmt.Println("Warning: With a stateless control plane, `deis logs` will be unavailable.")
+		fmt.Println("Additionally, components will need to be configured to use external persistent stores.")
+		fmt.Println("See the official Deis documentation for details on running a stateless control plane.")
```

Comment:
Yeah, I think that's a bit too transparent, though. We have no way of knowing which unit they're using. And I want @krancour to think about a better way to handle this other than mocking the unit (again, I think having controller ensure the log directory exists on startup is the way to go here), so I don't want to spend too much time worrying about the mock unit.